pkg/api/meta/help.go

```diff
@@ -39,6 +39,14 @@ func GetItemsPtr(list runtime.Object) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	// if we're a runtime.Unstructured, check to see if we have an `items` key
+	if unstructured, ok := list.(*runtime.Unstructured); ok {
+		if items, ok := unstructured.Object["items"]; ok {
+			return items, nil
```

Comment:
`items` will be a slice, not a pointer to a slice (according to `runtime.Unstructured`'s definition)